spec/dummy/config/application.rb

```diff
@@ -3,19 +3,14 @@
 require_relative 'boot'
 
 require 'rails'
-# Pick the frameworks you want:
 require 'active_model/railtie'
 require 'active_job/railtie'
-# require "active_record/railtie"
-# require "active_storage/engine"
 require 'action_controller/railtie'
 require 'action_mailer/railtie'
 require 'action_view/railtie'
-# require "action_cable/engine"
-# require "sprockets/railtie"
-# require "rails/test_unit/railtie"
 
 Bundler.require(*Rails.groups)
+# TODO: why is this necessary?
```

Comment:
The Dummy App doesn't require 'europeana/feedback_button' in a gemfile or elsewhere.